### PR TITLE
Adds compat data for AnimationEffectTiming

### PR DESCRIPTION
**Pull request by bennypowers**

### CODE REVIEW

File: api/AnimationEffectTiming.json [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+          },
+          "edge": {
+            "version_added": null
+          },
+          "edge_mobile": {
+            "version_added": false
+          },
+          "firefox": {
+            "version_added": "48",
+            "notes": "The Web Animations API is only enabled by default in Firefox Developer Edition and Nightly builds. You can enable it in beta and release builds by setting the preference <code>dom.animations-api.core.enabled</code> to <code>true</code>, and can disable it in any Firefox version by setting this preference to <code>false</code>."
```
Comment (teoli2003) [inline api/AnimationEffectTiming.json]: We use the "flags" directive to indicate that support (in release) is behind a flag. See the [docs](https://github.com/mdn/browser-compat-data/blob/master/schemas/compat-data-schema.md#flags) for more info and an example of use.
In this case, we also have to reformulate the note to say that the preference is on by default in Firefox Developer Edition and Nightly builds.

File: api/AnimationEffectTiming.json [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+          },
+          "firefox_android": {
+            "version_added": "48",
+            "notes": "The Web Animations API is only enabled by default in Firefox Developer Edition and Nightly builds. You can enable it in beta and release builds by setting the preference <code>dom.animations-api.core.enabled</code> to <code>true</code>, and can disable it in any Firefox version by setting this preference to <code>false</code>."
```
Comment (teoli2003) [inline api/AnimationEffectTiming.json]: We use the "flags" directive to indicate that support (in release) is behind a flag. See the [docs](https://github.com/mdn/browser-compat-data/blob/master/schemas/compat-data-schema.md#flags) for more info and an example of use.
In this case, we also have to reformulate the note to say that the preference is on by default in Firefox Developer Edition and Nightly builds.

**Comment by bennypowers** [comment]
I'm unclear as to the correct wording for the note, assuming there's also a flags directive. Can you link to an example?

**Comment by teoli2003** [comment]
Sure, here is the docs with an example: https://github.com/mdn/browser-compat-data/blob/master/schemas/compat-data-schema.md#flags

the "notes" could be something like: "The preference is set to \<code>true\</code> by default on Firefox Nightly and on Firefox Developer Edition, but not on the official release."

File: api/AnimationEffectTiming.json (hunk 2 above)
Comment (bennypowers) [reply api/AnimationEffectTiming.json]: Please see latest diff

**Pull request merged**